### ISSUE TITLE
FR: Apply classes for CSS

### ISSUE THREAD

**Issue by chrisgrieser** [scambier/obsidian-omnisearch#8]
Right now, the plugin does not add any html classes, so the prompt simply inherits what the styling the themes do for prompts (= Quick Switcher / Command Palette). 

However, as search is quite a different use case from selecting commands/files, it would make sense to style the the modal differently. To be able to do that, the plugin's html needs some plugin-specific css-classes.

<img width="779" alt="Screenshot 2022-04-10 22 58 53" src="https://user-images.githubusercontent.com/73286100/162639681-27a95ce3-1550-47e7-ab84-3f11e9a04cb8.png">

**Comment by scambier** [comment]
Is your feature request to _just_ add a few classes, or to customize the modal? [I just added a few classes](https://github.com/scambier/obsidian-omnisearch/commit/8bc57c4362d5ae41c7b810e7bbacc8de73a696c3) if anyone wants to tweak it.

> However, as search is quite a different use case from selecting commands/files (...)

I kinda like how it looks actually :)

Besides, the layout will likely change as I update the plugin and add new features, so I don't plan to spend time working on the style for now 👍

**Comment by chrisgrieser** [comment]
Thanks! Well, one think now possible is to target the omnisearch modal by with Supercharged Links, to add its icons to the search hits. (I did [a quick PR for that](https://github.com/mdelobelle/obsidian_supercharged_links/pull/102).)

<img width="363" alt="Screenshot 2022-04-13 13 38 44" src="https://user-images.githubusercontent.com/73286100/163172189-3db64f23-9260-4937-85a0-8f47e3c33193.png">

---

> I kinda like how it looks actually :)

Well, as a theme dev I do have some ideas 😆 
For example, I added some css to make the modal bigger. While the Command Palette and the Quick Switcher have less content to display, the search results from omnisearch actually are more extensive. This makes a bigger modal quite useful, in my view.

<img width="1468" alt="Screenshot 2022-04-13 13 41 00" src="https://user-images.githubusercontent.com/73286100/163172502-a052c86c-a328-4f26-971d-62d5eeb9a22b.png">

---

Also, Im gonna apply some theme-specific styling, but yeah, that's quite subjective. 🙂  Anyway, thanks for the quick implementation!
